devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -234,6 +234,14 @@ def configure() {
 	commands << zwave.multiChannelV3.multiChannelEndPointGet()
 	commands + getRefreshCommands()
 
+	// 1% is default Minimum dimming value for dimmers,
+	// when device is set to 1% - it turns off and device does not send any level reports
+	// Minimum dimming value has to be set to 2%, so the device's internal range would be 2-100%
+	// Still, for users it will relatively be 1-100% on the UI and device will report it.
+	// Parameter no. 60 – Minimum dimming value
+	commands << zwave.configurationV2.configurationSet(scaledConfigurationValue: 2, parameterNumber: 60, size: 1)
+	commands + getReadConfigurationFromTheDeviceCommands()
```

Comment:
did you mean to do `+=` here?

Reply:
> did you mean to do `+=` here?

@greens 
Yes, thanks for pointing that out!